### DISCUSSION TITLE
Can we have a separate host that can connect into shadow to allow analytics (on the switch) to catch the Tor traffic running through it

### DISCUSSION THREAD

**Discussion by stpsmith** [shadow/shadow#695] [Q&A] [answered]
**State your question concisely**

**Describe your current understanding of Shadow as it relates to your question**

**Describe which parts you are unsure about**

**Comment by robgjansen** [comment, marked as answer]
For now, you can achieve this using the shadow.config.xml file. In particular, you can add the 'logpcap' attribute to the host elements of the hosts you want to trace:

https://github.com/shadow/shadow/wiki/3.1-Shadow-Config#the-host-element

Here is the documentation for the 'logpcap' attribute:

> logpcap is a case insensitive boolean string (e.g. "true") that specifies that Shadow should log all network input and output for this node in PCAP format (for viewing in e.g. wireshark). pcapdir is the directory to which the logs should be saved for this host.
